packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -455,9 +494,11 @@ SwipeableDrawer.defaultProps = {
   disableDiscovery: false,
   disableSwipeToOpen:
     typeof navigator !== 'undefined' && /iPad|iPhone|iPod/.test(navigator.userAgent),
+  hysteresis: 0.6,
```

Comment:
Maybe 0.55 would be more appropriate

Reply:
I think, since minFlingThreshold is implemented it doesn't matter that much?! 0.5 would be a good value too

Reply:
@jniclas 0.5 is too low. It's not how native drawers behave. You have to really cross the half of the width.

Reply:
I see. Than leave the default value as before 0.6